mmv1/third_party/tgc/tests/data/example_google_gke_hub_membership.json

```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "name": "//gkehub.googleapis.com/projects/{{.Provider.project}}/locations/us-central1-a/memberships/my-membership",
+        "asset_type": "gkehub.googleapis.com/Membership",
+        "ancestry_path": "{{.Ancestry}}/project/{{.Provider.project}}",
+        "resource": {
+            "version": "v1",
+            "discovery_document_uri": "https://gkehub.googleapis.com/$discovery/rest?version=v1",
+            "discovery_name": "Membership",
+            "parent": "//cloudresourcemanager.googleapis.com/projects/{{.Provider.project}}",
+            "data": {}
```

Comment:
Can we please add a bit more comprehensive test case such that the data block has some fields in it? This will enhance the unit test case and give us more confidence in our underlying converter logic.